daphne_service_utils/dapf/src/main.rs

```diff
@@ -431,6 +452,50 @@ async fn main() -> Result<()> {
             println!("ID of the new HPKE config: {hpke_config_id}");
             Ok(())
         }
+        Action::Aggregate {} => {
+            let t = dapf::acceptance::Test::from_env()?;
+
+            tracing::info!("using vdaf: {:?}", t.vdaf_config);
+
+            let res = t
+                .test_helper(
+                    &TestOptions {
+                        bearer_token: t.leader_bearer_token.clone(),
+                        ..Default::default()
+                    },
+                    if t.helper_url.path().contains("/v02") {
```

Comment:
There is already a function for resolving the `DapVersion` for a given path. I'm not sure off hand what it's called.